src/components/Content/EventsList.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Link } from 'redux-little-router'
+
+import EventModel from '../../endpoints/models/EventModel'
+import Caption from './Caption'
+
+import style from './EventsList.css'
+
+import EventPlaceholder1 from '../../components/Content/assets/EventPlaceholder1.jpg'
+import EventPlaceholder2 from '../../components/Content/assets/EventPlaceholder2.jpg'
+import EventPlaceholder3 from '../../components/Content/assets/EventPlaceholder3.jpg'
+
+class Event extends React.Component {
```

Comment:
This looks very similar to Event.js. Are there major differences?

Reply:
you mean the Placeholder logic? nope, exactly the same, should it be excluded?

Reply:
I meant the whole Event component. Because there is another one in Event.js. But it's a bit different. Maybe rename this to EventLink?